sky/shell/platform/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -2,46 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "base/command_line.h"
-#include "base/trace_event/trace_event.h"
-#include "sky/shell/platform/ios/FlutterAppDelegate.h"
+#include "sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h"
 #include "sky/shell/platform/ios/framework/Headers/FlutterViewController.h"
-#include "sky/shell/switches.h"
 
-NSURL* URLForSwitch(const char* name) {
-  auto cmd = *base::CommandLine::ForCurrentProcess();
-  NSUserDefaults* defaults = [NSUserDefaults standardUserDefaults];
+@implementation FlutterAppDelegate {
+  UIWindow* _window;
+}
 
-  if (cmd.HasSwitch(name)) {
-    auto url = [NSURL fileURLWithPath:@(cmd.GetSwitchValueASCII(name).c_str())];
-    [defaults setURL:url forKey:@(name)];
-    [defaults synchronize];
-    return url;
-  }
+- (void)dealloc {
```

Comment:
Style-wise, I always see this as the last method defined in the file, but the style guide doesn't have anything to say on the subject, so not sure if it matters.  Chinmay?